recipes-domu/domu-image-android/files/meta-xt-images-extra/recipes-extended/android/android.bb

```diff
@@ -36,7 +36,18 @@ do_compile() {
     USRBINPATH_NATIVE="${RECIPE_SYSROOT_NATIVE}/usr/bin"
 
     # run Android build in sane environment
-    env -i HOME="$HOME" LC_CTYPE="${LC_ALL:-${LC_CTYPE:-$LANG}}" USER="$USER" \
+    if [ -z ${DDK_UM_PREBUILDS+x} ]; then
```

Comment:
We have discussed a possibility to avoid this "if" section if we turn `DDK_KM_PREBUILT_MODULE="${DDK_KM_PREBUILT_MODULE}"` definitions into something like:
1. MY_VAR = prebuilt ? DDK_KM_PREBUILT_MODULE="${DDK_KM_PREBUILT_MODULE}" : ""
2. Use ${MY_VAR} in bash code

Reply:
@andr2000 the param_maker class has been added to evaluate the variable and create the parameter